common_scripts/common.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04 as base
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    python3-pip \
+    curl \
+    python-dev \
+    git \
+    g++ \
+    make \
+    && curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+    && curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends nodejs yarn \
```

Comment:
Probably don't need nodejs and yarn just to build the python package

Reply:
Correct. Fixed in 763d51448f363a84b68798be066f8991fbbce4a0